MANIFEST.in

```diff
@@ -1,9 +1,8 @@
-global-exclude *.py[co]
-global-exclude __pycache__
 include .reuse/dep5
 include CHANGELOG.md
-include LICENSE.txt
+include LICENSE
 include requirements.txt
-include requirements-dev.txt
```

Comment:
Do we really want to exclude the dev requirements? This would keep people from running the tests due to missing deps. However we should enable users to run tests

Reply:
Sorry, you are absolutely right. And it was me arguing for it first :man_facepalming: 

It's readded.